Errata.md

```diff
@@ -11,6 +11,7 @@ The following corrections will be made in the second printing. (To determine whi
 
 Found by            | Chapter     | Page        | Correction                                                                                                                                                       
 ------------------- | ----------- | ----------- | -----------------------------------------------------------------------------------------------------------------------------------------------------------------
+Salim Gangji        | 1           | 12          | First paragraph: For a single executable, append <s>`/p:PublishSingleFile=true`</s> `-p:PublishSingleFile=true` to the command
```

Comment:
The change suggested from a /p: to a -p:
Both work, however the -p: is an improvement as it is more universal, not working only in windows as the /p: does